packages/rbac/v2/tests/integration/client/index.ts

```diff
@@ -0,0 +1,113 @@
+import { expect } from '@jest/globals';
+// import all the workspaces operations
+import WorkspacesRead from '../../../WorkspacesRead';
+import WorkspacesList from '../../../WorkspacesList';
+import WorkspacesCreate from '../../../WorkspacesCreate';
+import WorkspacesDelete from '../../../WorkspacesDelete';
+import WorkspacesPatch from '../../../WorkspacesPatch';
+import WorkspacesUpdate from '../../../WorkspacesUpdate';
+
+import { AxiosRequestConfig, AxiosResponse } from 'axios';
+import { WorkspacesWorkspace, WorkspacesWorkspaceListResponse } from '../../../api';
+import { WorkspacesPatchWorkspaceRequest, WorkspacesUpdateWorkspaceRequest, WorkspacesWorkspaceTypesQueryParam } from '../../..';
+
+import axios from 'axios';
+
+import { APIFactory } from '@redhat-cloud-services/javascript-clients-shared/utils';
+import { Configuration } from '@redhat-cloud-services/javascript-clients-shared/configuration';
+
+export const WORKSPACES_API_BASE = 'http://localhost:3000/api/rbac/v2';
+export const config: Configuration = new Configuration();
+const axiosInstance = axios.create();
+
+export const composedWorkspacesApi = APIFactory(
+  WORKSPACES_API_BASE,
+  {
+    WorkspacesRead: WorkspacesRead,
+    WorkspacesList: WorkspacesList,
+    WorkspacesCreate: WorkspacesCreate,
+    WorkspacesDelete: WorkspacesDelete,
+    WorkspacesPatch: WorkspacesPatch,
+    WorkspacesUpdate: WorkspacesUpdate,
+  },
+  { axios: axiosInstance, configuration: config },
+);
+
+// --- convenience wrappers for workspace operations ---
+
+export const createWorkspace = async (name: string, description: string, config?: AxiosRequestConfig) => {
+  // @ts-ignore
```

Comment:
@catastrophe-brandon to follow up and move assertions to the tests themselves and remove ts-ignores (fix issues).